From 16d027f9687038f801ff2f6f62ad4f18523dc842 Mon Sep 17 00:00:00 2001
From: Martin Pitt <martin.pitt@ubuntu.com>
Date: Mon, 5 Sep 2011 15:33:25 +0200
Subject: [PATCH 17/18] Sandisk Sansa Fuze: Set FolderSeparator

As per William Witt, the Sansa Fuze requires DOS folder separators in playlist
files.

https://bugs.freedesktop.org/show_bug.cgi?id=40441
---
 media-players/sandisk_sansa-fuze.mpi |    1 +
 1 files changed, 1 insertions(+), 0 deletions(-)

diff --git a/media-players/sandisk_sansa-fuze.mpi b/media-players/sandisk_sansa-fuze.mpi
index 6769e47..3d942b7 100644
--- a/media-players/sandisk_sansa-fuze.mpi
+++ b/media-players/sandisk_sansa-fuze.mpi
@@ -10,6 +10,7 @@ OutputFormats=audio/mpeg;audio/x-ms-wma;audio/audible;application/ogg;audio/ogg;
 
 [Playlist]
 Formats=audio/x-mpegurl
+FolderSeparator=\
 
 [storage]
 RequiresEject=true
-- 
1.7.6

